lib/package/commands.coffee

```diff
@@ -22,6 +22,14 @@ module.exports =
         @withInk ->
           boot()
           juno.runtime.evaluation.eval(move: true)
+      'julia-client:run-cell': =>
+        @withInk ->
+          boot()
+          juno.runtime.evaluation.eval(cell: true)
+      'julia-client:run-cell-and-move': =>
+        @withInk ->
+          boot()
+          juno.runtime.evaluation.eval(cell: true, move: true)
```

Comment:
If you implement it like this, the cursor will move to the end of the next *block*, not of the next *cell* like one would probably expect.